includes/Rocket.h

```diff
@@ -137,6 +150,13 @@ class Rocket {
 
     std::array<double, 9> I_{};  // Rocket moment of inertia tensor
 
+    // The following are in Geocentric frame
+    Vector3d launch_ecef_{150992.99, -4882549.85, 4087626.55};
+    Vector3d launch_geod_{40.111801, -88.228691, 216};
+    Vector3d r_geod_{40.111801, -88.228691,
+                     216};  // (40.111801, -88.228691, 216) - Talbot Laboratory
+    Vector3d r_ecef_{150992.99, -4882549.85, 4087626.55};
```

Comment:
why does rocket store its current location in geodesic coordinates and ecef coordinates. It should only store 1 of them and then derive the other from the conversion functions.